modules/extensions-xml-support/src/test/resources/modules/module-json-custom-types-catalog.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<types:mule xmlns="http://www.mulesoft.org/schema/mule/core"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:types="http://www.mulesoft.org/schema/mule/types">
+<catalogs xmlns="http://www.mulesoft.org/schema/mule/types" >
 
-    <types:catalog>
-        <types:type name="Type1" location="./module-json-custom-types-type1-schema.json"/>
-        <types:type name="Type2" location="./module-json-custom-types-type2-schema.json"/>
-    </types:catalog>
+    <catalog name="Type1" format="application/json">
+        <schema format="application/json+schema" location="./module-json-custom-types-type1-schema.json" />
```

Comment:
no purpose, just a way to make clear it's a local reference. Without the `./` works fine, as it relies in the plugin classloader to look for resources.